sam-react-app/coreconcept/src/moviesByGenre/MoviesByGenre.tsx

```diff
@@ -0,0 +1,37 @@
+import React, {Component, ChangeEvent} from "react";
+
+class MoviesByGenre extends Component {
+    state = {
```

Comment:
Same as above in Counter component: constructor is missing.

Reply:
done